packages/rocketchat-api/server/v1/groups.js

```diff
@@ -729,6 +729,24 @@ RocketChat.API.v1.addRoute('groups.setType', { authRequired: true }, {
 	},
 });
 
+RocketChat.API.v1.addRoute('groups.setAnnouncement', { authRequired: true }, {
```

Comment:
should we reduce code duplication on this routes (groups, channels, im) using the same functions?